src/clr-angular/forms/datepicker/date-container.spec.ts

```diff
@@ -70,6 +131,7 @@ export default function() {
       });
 
       it('projects the date input', () => {
+        context.detectChanges();
```

Comment:
why is this extra change detector here? Is it because it relies on the previous test?

Reply:
This is an artifact before I moved the newFormsLayout bit to a beforeEach, can be removed.